.pre-commit-hooks.yaml

```diff
@@ -183,3 +183,13 @@
     entry: trailing-whitespace-fixer
     language: python
     types: [text]
+-  id: update-commit-msg
+   name: update-commit-msg
+   entry: update-commit-msg
+   # Update to your ticket matching regex
+   args: ['--regex=TICKET-[0-9]']
+   language: python
+   stages: [commit-msg]
+   additional_dependencies: ['six', 'invoke']
+   files: .git/COMMIT_EDITMSG
```

Comment:
this is unnecessary (and in some cases incorrect) -- pre-commit will make sure to handle passing the right filenames here.  In the latest version, neither `types` nor `files` is required so it'll just do the right thing

Reply:
I think the reason why i added it is, without it, it was having some side-effects and it was updating files all changed files.

Reply:
hmm, probably before you added `stages: [commit-msg]`?

Reply:
yeah probably, i cannot reproduce it now. will remove.